test/serializer/basic/CircularFuncs.js

```diff
@@ -0,0 +1,32 @@
+// serialized function clone count: 0
```

Comment:
Is this testing additional function? If so, I was expecting some annotations to specify target additional function.